### PR TITLE
Remove duplicated function and unify it in compat

### PR DESCRIPTION
**Pull request by meetmangukiya**
https://github.com/coala-analyzer/coala/issues/2743

### CODE REVIEW

**Comment by Asnelchristian** [comment]
uhm @ meetmangukiya why do you have two commits?  btw. you have some issues with gitmate-bot, trying running `coala` to solve it ..

**Comment by meetmangukiya** [comment]
@makman2 :laughing: @meetmangukiya caught copying the solution proposed in the issue

**Comment by sils1297** [comment]
Can you rebase and fix gitmate issues?

**Comment by meetmangukiya** [comment]
@sils1297 I think I need some help, I don't understand why the other 3 commits are appearing in the PR, while I assume they are in the master branch already :/ ?

**Comment by sils1297** [comment]
Hi,

please take a look at those rebase instructions, that should get rid of it: [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala-analyzer/coala.git
```

(Be sure to use coala-bears on the coala-bears repository :))

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

**Comment by meetmangukiya** [comment]
Why did the size label change `M ->  S -> XS`?

**Comment by Makman2** [comment]
I don't know :O if you want I can manually mark it as `S` :P :P

File: coalib/misc/Compatability.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import json
+try:
+    JSONDecodeError = json.decoder.JSONDecodeError
+    JSONDecodeError = json.decoder.JSONDecodeError
```
Comment (Makman2) [inline coalib/misc/Compatability.py]: duplicate

**Comment by Makman2** [comment]
unack 5b0897e

**Comment by Makman2** [comment]
I would also change commit message, as you don't unify something here in this commit:
```
Introduce compatability module
```
You could additionally describe inside the body that this is needed from coala-bears :+1:

File: coalib/misc/Compatability.py (hunk 1 above)
Comment (meetmangukiya) [inline coalib/misc/Compatability.py]: I didn't notice when did that happen :fearful:

**Comment by Makman2** [comment]
ack d38504e

**Comment by meetmangukiya** [comment]
Sorry for that @Makman2 , actually I forgot to include the coala-bears part in the commit, that's why updated it again.

**Comment by Makman2** [comment]
ah you did change again :3 sure no problem, if something's left to be done don't hesitate, this time I was just fast enough :)

**Comment by meetmangukiya** [comment]
I'll update the commit title and description in the coala-bears PR as well :+1:

**Comment by Makman2** [comment]
cool :)
though after merge you need to add / prepend a commit to your coala-bears PR, as you need to update coala there to get your newest changes.

**Comment by Makman2** [comment]
ack d6be50b

**Comment by meetmangukiya** [comment]
Not really, I think the existing commit suits them(coala-bears PR) well @Makman2 what do you think?

**Comment by Makman2** [comment]
@rultor merge

**Comment by Makman2** [comment]
@rultor merge

**Comment by sils1297** [comment]
@rultor merge

**Comment by meetmangukiya** [comment]
Rultor was able to merge, when @sils1297 said so, rultor is interesting :/

**Comment by sils1297** [comment]
I changed some configuration

**Pull request merged**